webapi/src/main/twirl/queries/sparql/v1/createNewResource.scala.txt

```diff
@@ -20,30 +20,16 @@
 
```

Comment:
* Rename this `createNewResources.scala.txt`.
* Add a Scaladoc comment.
* Rename `ownerIri` to `creatorIri`.